### PR TITLE
[MDS-6115] Remove municipal wastewater from purpose and auth

### PR DESCRIPTION
**Pull request by asinn134**
## Objective 

[MDS-6115](https://bcmines.atlassian.net/browse/MDS-6115)

- Removing Municipal Wastewater option from Environmental Management Act in the Purpose and Authorization section.

### CODE REVIEW

File: services/common/src/tests/mocks/dataMocks.tsx [hunk 1]
```diff
@@ -8062,7 +8062,7 @@ export const PROJECT_SUMMARY_AUTHORIZATION_TYPES_TRANSFORMED = [
       { code: "AIR_EMISSIONS_DISCHARGE_PERMIT", description: "Air emissions discharge permit" },
       { code: "EFFLUENT_DISCHARGE_PERMIT", description: "Effluent discharge permit" },
       { code: "REFUSE_DISCHARGE_PERMIT", description: "Refuse discharge permit" },
-      { code: "MUNICIPAL_WASTEWATER_REGULATION", description: "Municipal Wastewater Regulation" },
```
Comment (matbusby-fw) [inline services/common/src/tests/mocks/dataMocks.tsx:8065]: Personally I prefer removing these kinds of things and re-adding them in the future.  Comments have a habit of staying in the code long past where we remember why they're there.  Not a huge deal, but who knows when we'll add this functionality back in.

**Review by matbusby-fw — CHANGES_REQUESTED** [review]
Looks great!  Just a small not on removing lines from the data mocks rather than commenting them out.

File: services/common/src/tests/mocks/dataMocks.tsx (hunk 1 above)
Comment (asinn134) [reply services/common/src/tests/mocks/dataMocks.tsx:8065]: Sounds good, the code that was commented out is now removed.

File: services/common/src/components/projectSummary/AuthorizationsInvolved.tsx [hunk 2]
```diff
@@ -185,10 +191,10 @@ const RenderEMAPermitCommonSections = ({ code, isAmendment, index }) => {
           <Alert
             description={
               <>
-                If yes, please attach a <b>letter with rationale</b> to support this exemption at{" "}
-                <b>Document Upload</b> section. Please note that requests may not always be granted.
-                Incomplete applications may be returned if they don&apos;t meet Ministry
-                requirements and the application fee may not be refunded.
+                <b>If yes, a final application form is required in the supporting document below</b>
+                . Note that exemptions aren&apos;t guaranteed. Incomplete applications may be
```
Comment (taraepp) [inline services/common/src/components/projectSummary/AuthorizationsInvolved.tsx:195]: Sonarcloud doesn't like the `</b>` with the "." on the new line. 😂 
Probably eslint doing you wrong here (I'm assuming it's rendering correctly).

LGTM!

**Review by taraepp — APPROVED** [review]

**Review by simensma-fresh — APPROVED** [review]

**Review by matbusby-fw — APPROVED** [review]

**Pull request merged**